docs/documentation/release-notes.md

```diff
@@ -121,7 +121,12 @@ Specify in the corresponding release notes file:
 'Updated the Docker image to '
```

Comment:
If we have "Updated the" we don't need this one right?

Reply:
guess so, but this is just the documentation, do you want to remove it from here?

Reply:
Oh, I don't mind it then, your call

Reply:
Made the change, so it wont look weird